charts/ctlog/tv.yaml

```diff
@@ -0,0 +1,16 @@
+forceNamespace: ctlog-system
+server:
```

Comment:
i think this might also be a debugging file?

Reply:
doh.
removed